lib/interactive_viewer.ml

```diff
@@ -69,11 +69,29 @@ let view (patches : Patch.t list) =
     | Some z -> Lwd.var z
     | None -> failwith "zipper_of_list: empty list"
   in
+  let curr_scroll_state = Lwd.var W.default_scroll_state in
+  let change_scroll_state action state =
+    let off_screen = action = `Action && state.W.position > state.W.bound in
+    let scroll_on_output =
```

Comment:
What does `scroll_on_output` mean ? Why isn't `off_screen` enough to avoid scrolling too far ?